src/templates/community-champion-profile.js

```diff
@@ -257,6 +273,230 @@ const Bio = styled(P)`
+              {
+                  type: 'GitHub',
+                  url: socials.githubUrl,
+                  Icon: GithubIcon,
+              },
+              {
+                  type: 'Twitch',
+                  url: socials.twitchUrl,
+                  Icon: TwitchIcon,
+              },
+              {
+                  type: 'YouTube',
+                  url: socials.youtubeUrl,
+                  Icon: YoutubeIcon,
+              },
+              {
+                  type: 'Facebook',
+                  url: socials.facebookUrl,
+                  Icon: FacebookIcon,
+              },
+              {
+                  type: 'Twitter',
+                  url: socials.twitterUrl,
+                  Icon: TwitterIcon,
+              },
+          ]
+        : [];
+    return (
+        <div>
+            <LinksTitle collapse>Social</LinksTitle>
+            <SocialLinksContainer>
+                {links.map(
+                    ({ type, url, Icon }) =>
+                        url && (
+                            <SocialLink
+                                key={type}
+                                icon={
+                                    <Icon
+                                        height={size.default}
+                                        width={size.default}
+                                    />
+                                }
+                                type={type}
+                                link={url}
+                            />
+                        )
+                )}
+            </SocialLinksContainer>
+        </div>
+    );
+};
+
+const SocialLinkItemContainer = styled(Link)`
+    align-items: center;
+    column-gap: ${size.xsmall};
+    display: grid;
+    grid-template-columns: ${size.default} auto;
+    text-decoration: none;
+    path {
+        fill: ${({ theme }) => theme.colorMap.devWhite};
+    }
+    &:hover {
+        path {
+            fill: ${({ theme }) => theme.colorMap.darkGreen};
+        }
+    }
+`;
+
+const SocialLinksContainer = styled('div')`
+    column-gap: ${SOCIAL_LINKS_GRID_COLUMN_GAP};
+    display: grid;
+    grid-auto-flow: column;
+    grid-template-columns: repeat(2, max-content);
+    grid-template-rows: repeat(3, auto);
+    row-gap: ${size.xsmall};
+    @media ${screenSize.upToSmallDesktop} {
+        column-gap: ${size.mediumLarge};
+        grid-auto-flow: row;
+        grid-template-columns: repeat(3, max-content);
+        grid-template-rows: repeat(2, auto);
+    }
+    @media ${screenSize.upToSmall} {
+        grid-template-columns: repeat(
+            auto-fill,
+            minmax(${SOCIAL_LINK_MAX_WIDTH}, 1fr)
+        );
+        grid-template-rows: none;
+    }
+`;
+
+const SocialLinkText = styled(P2)`
+    @media ${screenSize.upToMedium} {
+        font-size: ${fontSize.small};
+        line-height: ${lineHeight.small};
+    }
+`;
+
+const SocialLink = ({ icon, link, type }) => (
+    <SocialLinkItemContainer href={link}>
```

Comment:
Should these links open in a new tab? (Links on this page in general)

Reply:
good question. let me ask Mike!

Reply:
we will open in a new tab!